### PR TITLE
[#167894707] Ship BOSH director logs to CloudWatch

### PR DESCRIPTION
**Pull request by tlwr**
[Story](https://www.pivotaltracker.com/n/projects/1275640/stories/167894707)

What
----

- We need to create some log groups for our audit logs
- We need to retain the data for sometime
- We need to pull in the awslogs release
- We need to configure the awslogs jobs on the BOSH director to ship logs

See https://github.com/alphagov/paas-aws-account-wide-terraform/pull/188

How to review
-------------

Code review

`make dev upload-cyber-tfvars`

Deploy it to your dev env

Check that the log groups exist and are subscribed to the destination

I've tested that bootstrap works

Who can review
--------------

Describe who can review the changes. Or more importantly, list the people
that can't review, because they worked on it.

### CODE REVIEW

File: terraform/globals.tf [hunk 1]
```diff
@@ -104,3 +104,17 @@ variable "assets_prefix" {
...
+
+variable "log_groups_to_ship_to_csls" {
+  description = "The names of the log groups to ship to CSLS, without the -env suffix"
+  type        = "list"
+
+  default = [
+    "bosh_d_audit",
+    "bosh_d_audit_worker",
+    "bosh_d_auth",
+    "bosh_d_credhub_events",
```
Comment (46bit) [inline terraform/globals.tf]: To match the rest of the PR:
```suggestion
    "bosh_d_credhub_security_events",
```

File: terraform/globals.tf [hunk 2]
```diff
@@ -104,3 +104,17 @@ variable "assets_prefix" {
   description = "Prefix for global assests like S3 buckets"
   default     = "gds-paas"
 }
+
+variable "log_groups_to_ship_to_csls" {
+  description = "The names of the log groups to ship to CSLS, without the -env suffix"
```
Comment (46bit) [inline terraform/globals.tf]:
```suggestion
  description = "The names of the log groups to ship to CSLS, without the _env suffix"
```

File: terraform/globals.tf (hunk 1 above)
Comment (tlwr) [reply terraform/globals.tf]: Good catch

Force pushed resolved: https://github.com/alphagov/paas-bootstrap/compare/98f70903da4d8835dd0e38ed8ebf3495428bc474..5d36376c7cf9d63d5594741230ed3fb87c5233cd

File: terraform/globals.tf (hunk 2 above)
Comment (tlwr) [reply terraform/globals.tf]: Good catch

Force pushed resolved: https://github.com/alphagov/paas-bootstrap/compare/98f70903da4d8835dd0e38ed8ebf3495428bc474..5d36376c7cf9d63d5594741230ed3fb87c5233cd

**Review by 46bit — APPROVED** [review]

**Pull request merged**